### PR TITLE
PEER-255 Add docker-compose config for user-db and user-service

### PR DESCRIPTION
**Pull request by SeeuSim**

### CODE REVIEW

File: backend/user/express.Dockerfile [hunk 1]
```diff
@@ -2,17 +2,18 @@ FROM node:lts-alpine AS build
...
 COPY . .
+COPY ".env.${env}" .env
 RUN npm run build
 
 FROM node:lts-alpine AS production
 WORKDIR /data/question-express
 COPY --from=build /data/question-express/package*.json ./
 RUN npm ci --omit=dev
 COPY --from=build --chown=node:node /data/question-express/dist ./dist
+COPY --from=build /data/question-express/.env .env
```
Comment (TabrizPlv) [inline backend/user/express.Dockerfile]: Copying `env` files in image may lead to security vulnerabilities.
Also, copying here is not necessary as you are inserting the `env` file in `docker-compose`

File: backend/user/express.Dockerfile [hunk 2]
```diff
@@ -2,17 +2,18 @@ FROM node:lts-alpine AS build
 WORKDIR /data/question-express
```
Comment (TabrizPlv) [inline backend/user/express.Dockerfile]: Replace all `question` with `user`

**Review by TabrizPlv — CHANGES_REQUESTED** [review]

File: backend/user/express.Dockerfile (hunk 1 above)
Comment (SeeuSim) [reply backend/user/express.Dockerfile]: it needs it for the `env-cmd` method, otherwise you can use `dotenv` to read from the environment itself

**Review by TabrizPlv — APPROVED** [review]
LGTM

**Pull request merged**